html.json

```diff
@@ -0,0 +1,55 @@
+{
+  "deliverables": [
+    {
+      "name": "HTML5",
+      "context": {
+        "inputs": ["userguide.ditamap"],
+        "profiles": {
+          "ditavals": ["resources/html.ditaval"]
+        }
+      },
+      "output": ".",
+      "publication": {
+        "transtype": "html5",
+        "params": [
+          {
+            "name": "args.copycss",
+            "value": "yes"
+          },
+          {
+            "name": "args.css",
+            "value": "dita-ot-doc.css"
+          },
+          {
+            "name": "args.csspath",
+            "value": "css"
+          },
+          {
+            "name": "args.cssroot",
+            "value": "resources"
+          },
+          {
+            "name": "args.gen.task.lbl",
+            "value": "YES"
+          },
+          {
+            "name": "args.hdr",
+            "href": "resources/header.xml"
```

Comment:
Using `href` to define the parameter value will resolve the value to an absolute URI before handing it to DITA-OT. This means `href` cannot be used for path or URI values that need to be relative when they are used by DITA-OT, for example `args.csspath` parameter must be a relative path.

Reply:
So how to pass such values that need to remain relative?

Reply:
With just `value`. Just like Ant's `value` and `location`.

Reply:
IMO this is not optimal, but as long we have Ant in between, this is all we can give the users.

Reply:
After the changes in dita-ot/dita-ot@5a6f380, it seems we now have 3 options for defining resource paths, each of which will be interpreted differently:

- `href` resolves to an absolute URI for things like `args.hdr` that require that
- `value` remains relative, for things like `args.csspath` that describe a relative path in output
- `file` may also be defined with a path relative to the project file (like `href`), but the result is expanded to an absolute system path like Ant's `@location` attribute, for things like `args.cssroot`